src/xokdinst.rs

```diff
@@ -246,7 +257,37 @@ enum Opt {
     },
 }
 
-fn cmd_installer(version: Option<&str>) -> std::process::Command {
+/// Compute master memory: host memory - 6GB (4 for worker + 2 for bootstrap)
+fn get_master_mem(libvirt_auto_size: bool) -> u64 {
+    let default = 4096;
+    if !libvirt_auto_size {
+        return default;
+    }
+    match mem_info() {
+        Err(e) => {
+            eprintln!("{}: Couldn't get host total memory, using 4GB for master", e);
```

Comment:
I think this will read better with the error after, like:
```suggestion
            eprintln!("Failed to get total host memory, falling back to {}MB: {}", default, e);
```